src/components/harnesses/HarnessAssembly.jsx

```diff
@@ -1,7 +1,67 @@
 import React from "react";
 
 const HarnessAssembly = () => {
-  return <div>HarnessAssembly</div>;
+  return (
+    <div>
+      <h1 className="font-Noto Sans HK flex justify-center text-transparent bg-clip-text bg-gradient-to-r from-jm-blue-300 via-jm-blue-200 to-jm-blue-100 text-[32px] mb-6 tracking-wide underline-offset-8 whitespace-break-spaces underline decoration-jm-blue-200 ">
+        <b>J&M Products Harness Assembly Overview </b>
+      </h1>
+      <div className="flex justify-center">
+        <ul className=" text-left text-[14px] list-disc flex flex-col font-Noto Sans HK w-5/12 tracking-wider">
```

Comment:
use a tailwind class instead pure pixels text-[14px]
Also the same thing with the font